relayer/types/types.go

```diff
@@ -24,17 +23,17 @@ type Message struct {
 	Data          []byte `json:"data"`
 	MessageHeight uint64 `json:"messageHeight"`
 	EventType     string `json:"eventType"`
+	key           *MessageKey
```

Comment:
The purpose of this struct is to have a plain message, adding key doesn't make sense...